packages/uui-symbol-lock/rollup.config.js

```diff
@@ -0,0 +1,6 @@
+import { UUIProdConfig } from '../rollup-package.config';
+
+export default UUIProdConfig({
+  entryPoints: ['index', 'uui-symbol-lock.element'],
```

Comment:
The only entrypoint should be 'index' on a package. You could probably redo the standard files using the latest plop templates